providers/attio/read.go

```diff
@@ -0,0 +1,44 @@
+// nolint
+package attio
+
+import (
+	"context"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/spyzhov/ajson"
+)
+
+var dummyNextPageFunc = func(*ajson.Node) (string, error) {
+	return "", nil
+}
+
+func (c *Connector) Read(ctx context.Context, config common.ReadParams) (*common.ReadResult, error) {
+	if err := config.ValidateParams(true); err != nil {
+		return nil, err
+	}
+
+	if !supportedObjectsByRead.Has(config.ObjectName) {
+		return nil, common.ErrOperationNotSupportedForObject
+	}
+
+	url, err := c.getApiURL(config.ObjectName)
+	if err != nil {
+		return nil, err
+	}
+
+	url.WithQueryParam("limit", "1")
```

Comment:
This doesn't seem right. This is correct during development process. We don't want to query only 1 item at a time!
Also if some objects have offset pagination support that should be implemented instead of `dummyNextPageFunc`.